src/main/java/duke/Time.java

```diff
@@ -0,0 +1,62 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import duke.exception.DukeException;
+
+/**
+ * Time class is used to parse the date and time from
+ * {@code String} to {@code LocalDateTime} or {@code LocalDate} object.
+ */
+public class Time {
+
+    private static final String DATE_FORMAT = "d/M/yyyy";
+    private static final String TIME_FORMAT = "HH:mm";
+    private static final String DATE_DISPLAY_FORMAT = "d MMM yyyy";
+    private static final String TIME_DISPLAY_FORMAT = "h:mma";
+    public static final DateTimeFormatter DATE_FORMATTER =
+            DateTimeFormatter.ofPattern(DATE_FORMAT);
+    public static final DateTimeFormatter DATE_TIME_FORMATTER =
+            DateTimeFormatter.ofPattern(DATE_FORMAT + " " + TIME_FORMAT);
+    public static final DateTimeFormatter DATE_TIME_DISPLAY_FORMATTER =
+            DateTimeFormatter.ofPattern(DATE_DISPLAY_FORMAT + ", " + TIME_DISPLAY_FORMAT);
+    public static final DateTimeFormatter DATE_DISPLAY_FORMATTER =
+            DateTimeFormatter.ofPattern(DATE_DISPLAY_FORMAT);
```

Comment:
Great that you have a separation of the formatting so that you can easily compose new formats